packages/vscode-ui5-language-assistant/test/suite/extension-spec.ts

```diff
@@ -78,7 +78,7 @@ describe("the Language Server Client Integration Tests", () => {
       "unified",
       "upload",
       "util",
-      "ux3",
+      "uxap",
```

Comment:
why did this change? it does not look like a filter has been applied?

Reply:
The test checks `includes.members` and not `equalsInAnyOrder`. `ux3` was removed from the result. `uxap` was always there, just not tested.
I think there is general room for improvements in the integration tests - instead of testing several completion use cases we can only check one, but we should check other scenarios too like hover and validation (at least as a smoke test), and maybe even a test specific for how settings affect the operations (e.g. check no deprecated APIs are returned by default, then turn the setting on and off and check again).